buddyboss/buddyboss-platform
# PROD-706 #3109 - Admin names should be display in message header section

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3109.

### How to test the changes in this Pull Request:

1. Go to message section
2. Create custom thread
3. Added multiple user with two or more admins should be there
4. Click on load more and see admins are not displaying on message header section

### Proof Screenshots or Video

https://screencast-o-matic.com/watch/cr6tD2VlDtw

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
